ec2/ec2-create-instance.sh

```diff
@@ -221,8 +221,12 @@ sudo yum -q -y install ansible git nano
 git clone -b $DA_BRANCH https://github.com/GlobalDataverseCommunityConsortium/dataverse-ansible.git dataverse
 export ANSIBLE_ROLES_PATH=.
 ansible-playbook $VERBOSE_ARG -i dataverse/inventory dataverse/dataverse.pb --connection=local $GVARG
+touch /tmp/ansible_complete
```

Comment:
I like this idea of the "ansible_complete" file but some of the steps above could fail, right? Should we separate all these commands with `&&` to give us confidence that they all succeeded?

Reply:
we set `-e` in the sh-bang, so in theory these failures should also cause the script to exit. it was the unexpected SSH termination we're working around.